power_grid_model_c/power_grid_model/include/power_grid_model/topology.hpp

```diff
@@ -293,15 +296,19 @@ class Topology {
         }
 
         // copy all the far-end non-cyclic node, in reverse order
+        // NOLINTNEXTLINE(modernize-use-ranges) // waiting for std::views
         std::copy_if(dfs_node_copy.crbegin(), dfs_node_copy.crend(), std::back_inserter(dfs_node),
                      [this](Idx x) { return node_status_[x] == -1; });
+
         // copy all cyclic node
         std::vector<Idx> cyclic_node;
+        // NOLINTNEXTLINE(modernize-use-ranges) // waiting for std::views
         std::copy_if(dfs_node_copy.cbegin(), dfs_node_copy.cend(), std::back_inserter(cyclic_node),
                      [this](Idx x) { return node_status_[x] == -2; });
 
         // reorder does not make sense if number of cyclic nodes in a sub graph is smaller than 4
         if (cyclic_node.size() < 4) {
+            // NOLINTNEXTLINE(modernize-use-ranges) // waiting for std::views
```

Comment:
in particular `std::views::reverse`

Reply:
see https://github.com/PowerGridModel/power-grid-model/compare/main...feature/use-std-views (which also depends on https://github.com/PowerGridModel/power-grid-model/pull/921)